spec/lib/models/file-spec.js

```diff
@@ -77,21 +77,6 @@ describe('Models.File', function () {
             });
         });
 
-        describe('md5', function() {
```

Comment:
A patch is needed to remove the md5 references from on-http and on-skupack and an accompanying patch to add sha256 computation to the microkernel before this can be removed.